assets/schema_input.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/mhcquant/master/assets/schema_input.json",
+    "title": "nf-core/mhcquant pipeline - params.input schema",
+    "description": "Schema for the file provided with params.input",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "ID": {
+                "type": "integer",
+                "errorMessage": "Provide an unique identifier for the replicate, must be a numeric value"
+            },
+            "Sample": {
+                "type": "string",
+                "pattern": "^\\S+-?",
+                "errorMessage": "Sample name must be provided and cannot contain spaces"
+            },
+            "Condition": {
+                "type": "string",
+                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
```

Comment:
This is checking that it ends with `fastq` which I guess it is not desired? PRobably the Regex should be the same as for the Sample above